kubernetes/pods.go

```diff
@@ -3,28 +3,42 @@ package kubernetes
 import (
 	"context"
 	"fmt"
-	"sort"
 	"sync"
 	"time"
 
 	log "github.com/sirupsen/logrus"
+	"github.com/zalando-incubator/kube-ingress-aws-controller/aws"
```

Comment:
this should not be a dependency in the kubernetes package
We should convert it into aws.CNIEnpoint in aws/adapter.go